test/abi.ts

```diff
@@ -0,0 +1,18 @@
+import { getABI } from "../src/utils/abi";
+import { assertPromiseError, expect } from "./utils";
+
+describe("ABI Utils", () => {
+  describe("getABI", async () => {
+    it("Retrieve Multicall3 ABI", async () => {
+      expect(await getABI("Multicall3"));
```

Comment:
Could we fuzz test for all the available contracts?

Reply:
My thinking was that it's OK just to test for 1 happy path and 1 sad path, to make sure those two cases work OK. We could iterate over each known contract and verify that it's readable, but then we'd have to maintain that list or load it dynamically via an fs operation. That might detect if someone saves a bad ABI, but they'd anyway catch that as soon as they tried to use it, right?

Reply:
I modified the test to read all files from the ABI dir and require that it doesn't throw when we read them. That should help to catch any bad files before they get merged :+1: 

https://github.com/across-protocol/sdk-v2/pull/420/commits/713d135b258f07ef6365e0f42a887c0413253d5a